### ISSUE TITLE
Selectpicker with data-style="btn-[...]" and title has poorly colored title

### ISSUE THREAD

**Issue by lselbeck** [snapappointments/bootstrap-select#1840]
**Expected:**
If you create a selectpicker with data-style="btn-primary" (or any other btn-), the color of the text on the button is white, to make it readable against the button's color.

**Problem:**
If you create a selectpicker with data-style="btn-primary" and title="whatever", the color of the text is a light grey that is hard to read against the button's color.

**Example:**
https://embed.plnkr.co/I42IkPUTss1EsLKeTZc1/

**Solution:**
Give selectpickers with data-style="btn-*" the same white color font regardless of whether it has a title or not.  To fix in my project, I just overrode the color to be white with a class.

**Comment by kamikazechaser** [comment]
This issue is still persistent and should be classified as a bug with unintended behavior. Allow users to easily customize the title. Temporary fix is to override the class, as in : 

```css
.bootstrap-select>.dropdown-toggle.bs-placeholder, 
.bootstrap-select>.dropdown-toggle.bs-placeholder:active, 
.bootstrap-select>.dropdown-toggle.bs-placeholder:focus, 
.bootstrap-select>.dropdown-toggle.bs-placeholder:hover { color: white; }
```

**Comment by caseyjhol** [comment]
This is fixed in v1.13.0+.
